mmap+memset+fork.c

```diff
@@ -27,6 +27,8 @@ void usage(){
 	printf("-e, --end               mmaps and memsets till 2^(arg)\n");
 	printf("-m, --map_size          mmaps given size\n");
 	printf("-n, --set_size          memsets given size\n");
+	printf("-u, --map_populate      sets MAP_POPULATE flag in mmap\n");
+	printf("-a, --map_anonymous     sets MAP_ANONYMOUS flag in mmap\n");
 	exit(EXIT_FAILURE);
```

Comment:
You have two choices here:  1) sort this entire list alphabetically or  2) group common options together (so these new map_xyz) elements could up after map_shared, map_private.